### ISSUE TITLE
Loading Feedback

### ISSUE THREAD

**Issue by KPull** [camilleriluke/ejja-ha-nieklu#23]
Show feedback on the app while data is still loading from the server.

**Comment by camilleriluke** [comment]
For some reason the connection with the database seems to be slower then before. 

We might need to rethink that area

**Comment by KPull** [comment]
Yep I noticed. I didn't change anything in serverside code though. Maybe it's the new Mongo driver?


Sent from my Samsung Galaxy smartphone.


-------- Original message --------

**Comment by camilleriluke** [comment]
I think it might be related.

The strange thing is that the server and the backend are both based in EU.

I'll send an email to the provider maybe we can get some feedback from them.
